public/app/plugins/datasource/elasticsearch/components/QueryEditor/MetricAggregationsEditor/utils.ts

```diff
@@ -112,7 +112,7 @@ export const metricAggregationConfig: MetricsConfiguration = {
     label: 'Moving Average',
     requiresField: true,
     isPipelineAgg: true,
-    versionRange: '>=2.0.0',
+    versionRange: '>=2.0.0, <8.0.0',
```

Comment:
I think there should be no comma:

```suggestion
    versionRange: '>=2.0.0 <8.0.0',
```

```javascript
> console.log(semver.satisfies('7.0.0', '>=2.0.0, <8.0.0'))
> false

> console.log(semver.satisfies('7.0.0', '>=2.0.0 <8.0.0'))
> true

```

Reply:
good catch!